atlassian_jwt_auth/contrib/django/decorators.py

```diff
@@ -1,5 +1,6 @@
 from functools import wraps
 
+import six
```

Comment:
That's fine. I haven't had to add it to a project before, it's usually already present, and I forgot about Django having a copy.